component-classes/index.js

```diff
@@ -398,13 +398,14 @@ export const input = {
 };
 
 export const select = {
-  default: 'block text-m mb-0 leading-m s-text s-bg s-border hover:s-border-hover active:s-border-active rounded-4 py-12 px-8 block border-1 w-full focusable focus:[--w-outline-offset:-2px] appearance-none pr-32 cursor-pointer caret-current',
-  disabled: 's-bg-disabled-subtle s-border-disabled hover:s-border-disabled! active:s-border-disabled! s-text-disabled pointer-events-none',
```

Comment:
I think we need to add `active:s-border-disabled!` to `backwardsCompatibleClasses`.